src/components/Common/ResourceCategoryForm.tsx

```diff
@@ -79,6 +72,16 @@ export function ResourceCategoryForm({
   const queryClient = useQueryClient();
   const isEditing = !!categorySlug;
 
+  const formSchema = z.object({
+    title: z.string().min(1, t("title_is_required")),
```

Comment:
```suggestion
    title: z.string().min(1, t("field_required")),
```

https://github.com/ohcnetwork/care_fe/blob/2778fba733eced09b0568ece60e5b5da382b47c1/public/locale/en.json#L2224